modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -156,27 +143,31 @@ Alternatively, run `rpk topic describe` again to show your reassigned replica se
-2          2       1      [1 2 3]   0                 0
+0          1       2      [1 2 3]   0                 645
+1          1       2      [1 2 3]   0                 682
+2          3       1      [1 2 3]   0                 672
 ----
-
-To cancel an in-progress partition reassignment with the Kafka API, use the flags `--cancel --preserve-throttles`:
-
++
+To cancel all in-progress partition reassignments, run `move-cancel`:
++
+[,bash]
+----
+rpk cluster partitions move-cancel
+----
++
+To cancel specific movements to or from a given node, run:
++
 [,bash]
 ----
-kafka-reassign-partitions.sh --bootstrap-server localhost:9092,localhost:9093,localhost:9094,localhost:9095 --reassignment-json-file example.json --cancel --preserve-throttles
-Successfully cancelled partition reassignments for: test-1,test-2
+rpk cluster partitions move-cancel --node 2
 ----
 
 === Differences in partition balancing between Redpanda and Kafka
 
-* Kafka's `kafka-reassign-partitions.sh` script attempts to use throttle configurations that Redpanda does not support, such as `replica.alter.log.dirs.io.max.bytes.per.second`. Include the flag `--preserve-throttles` to avoid errors when verifying or canceling a partition reassignment.
-* Kafka supports increasing and decreasing the topic replication factor through partition reassignments. Redpanda currently doesn't support this.
 * In a partition reassignment, you must provide the broker ID for each replica. Kafka validates the broker ID for any new replica that wasn't in the previous replica set against the list of alive brokers. Redpanda validates all replicas against the list of alive brokers.
-* When there are two identical partition reassignment requests, Kafka cancels the first one without returning an error code, while Redpanda rejects one with `unknown_server_error`.
+* When there are two identical partition reassignment requests, Kafka cancels the first one without returning an error code, while Redpanda rejects the second one with `Partition configuration update in progress` or `update_in_progress`.
 * In Kafka, attempts to add partitions to a topic during in-progress reassignments result in a `reassignment_in_progress` error, while Redpanda successfully adds partitions to the topic.
-* Kafka doesn't support shard-level partition assignments, but Redpanda does. When resolving a partition reassignment, Redpanda automatically determines the shard placements. If you want a partition on a specific shard, you must assign partitions with the Admin API.
+* Kafka doesn't support shard-level partition assignments, but Redpanda does. If you want to specify a shard for partition assignments, for example run `rpk cluster partitions move test -p 1:3-0,1-0,2-0 -p 2:2-1,1-1,3-1` where all replicas of partition `1` will be assigne on shard `0` on each broker and all replicas of partition `2` will be assigned on shard `1` on each broker. See more details in `rpk cluster partitions move --help`.
```

Comment:
In 24.1 we are going to get rid of manual shard assignments and let redpanda choose the shard automatically in all cases. So I wouldn't go into too much detail about this.

Reply:
I'm not sure this functionality has practical value today (were there cases where we or customers had to use it?) and because hopefully it will disappear soon, I'd rather not advertise it today.

Reply:
updated as per our DM, thanks!